scripts/cti

```diff
@@ -257,6 +257,7 @@ sed -e "s/{pod_name}/${pod_name}/g" \
     ${DIR}/cluster_test_pod_template.yaml > ${specfile}
 
 echo "Using cluster : ${WORKSPACE}"
+echo 'If connection aborts, use "kubectl logs ${WORKSPACE}" to fetch logs'
```

Comment:
mmm, did you check if this works?

Reply:
this is a great question, no I didn't. Will check it

Reply:
Ok. Please do test PRs before sending them out.